featurebyte/api/base_table.py

```diff
@@ -68,6 +68,27 @@ def info(self) -> ColumnInfo:
         column_info = next(col for col in self.parent.columns_info if col.name == self.name)
         return cast(ColumnInfo, column_info)
 
+    @property
+    def cleaning_operations(self) -> List[CleaningOperation]:
+        """
+        Cleaning operations applied to the column of the table.
+
+        Returns
+        -------
+        List[CleaningOperation]
+            List of cleaning operations applied to the column of the table.
+
+        Examples
+        --------
+        >>> event_table = catalog.get_table("GROCERYINVOICE")
+        >>> event_table["GroceryInvoiceGuid"].cleaning_operations
+        []
+        """
+        for column_cleaning_operations in self.parent.column_cleaning_operations:
```

Comment:
do we need the parent check here?

Reply:
I don't think parent check is required here since we all the table column should have parent and we don't allow table column transformation.